samples/SampleApp/Gluon.targets

```diff
@@ -1,10 +1,11 @@
+<!-- Sample targets file. See SampleApp.fsproj for example of how to hook it into MSBuild -->
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <GluonToolPath>$(MSBuildProjectDirectory)\..\..\src\Gluon.CLI\bin\$(Configuration)\Gluon.CLI.exe</GluonToolPath>
     <GluonOutputPath>bin\Generated.ts</GluonOutputPath>
   </PropertyGroup>
   <Target Name="GluonCompiler" AfterTargets="Build">
-    <Exec Command="&quot;$(GluonToolPath)&quot; --out &quot;$(GluonOutputPath)&quot; --reflect &quot;$(TargetPath)&quot;"
+    <Exec Command="mono &quot;$(GluonToolPath)&quot; --out &quot;$(GluonOutputPath)&quot; --reflect &quot;$(TargetPath)&quot;"
```

Comment:
Not good for cross-platform builds. How do you toggle whether or not to use `mono` within `MSBuild`?